teams/types.toml

```diff
@@ -0,0 +1,26 @@
+name = "types"
+subteam-of = "compiler"
```

Comment:
can we leave a comment here mentioning that this is both a subteam of lang and compiler, but filed under compiler due to technical reasons?